cypress/integration/us-0000-acceptance-test-spec.js

```diff
@@ -16,10 +16,10 @@ describe('CGM automatic import handling', () => {
         cy.get('[data-test=input-filename]').should('be.empty')
         cy.get('[data-test=input-latest-modification]').should('be.empty')
 
-        ftp.runOnFtp(ftpUser, ftpPassword, () => {
+        ftp.runOnFtp(fbUser, fbPassword, () => {
             ftp.copyFileToFtp('us-0000/20210701.zip', '/2021/07/01');
```

Comment:
We should replace this line by ftp.copyFileToFtp('us-0000/20210701.zip', 'ftp/2021/07/01'); to be in line with https://github.com/farao-community/gridcapa-deployment/pull/18